### PR TITLE
Fix/ms 579/highlighter support in text entry inline interaction

### PR DESCRIPTION
**Pull request by AnastasiaArcadia**
Related to: https://oat-sa.atlassian.net/browse/MS-579

**Changes**

In text entry inline interactions `input` element was changed to `div` for `reviewRenderer` to support highlight plugin.

**How to test?**

- Create a test with text entry inline interaction
- Take the test on tao
- Login as a scorer on MS to review the responses
- All the text entry inline interactions should be displayed as expected and should have been rendered in a `div` instead of `input`

**Depended on**
- [x] Release and merged https://github.com/oat-sa/tao-item-runner-qti-fe/pull/121

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
     },
     "dependencies": {
         "@oat-sa/tao-item-runner": "0.6.1",
-        "@oat-sa/tao-item-runner-qti": "0.14.1"
+        "@oat-sa/tao-item-runner-qti": "git://github.com/oat-sa/tao-item-runner-qti-fe.git#fix/MS-579/highlighter-support-in-text-entry-inline-interaction"
```
Comment (andrey-ponamarev) [inline views/package.json:12]: - [ ]  Please, don't forget to change the version

**Review by andrey-ponamarev — APPROVED** [review]
- [x] ⚠️ Required changes in package*.json
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by ampaveliev — APPROVED** [review]
Code review only. Looks fine

**Review by andrey-ponamarev — CHANGES_REQUESTED** [review]
Please fix conflicts and update version in package*.json

**Review by andrey-ponamarev — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**